PhoneFormat.js

```diff
@@ -192,15 +192,14 @@ Phoneformat.exampleMobileNumber = function (country) {
  * @param {String} phone
  */
 Phoneformat.cleanPhone = function (phone) {
-  phone = phone.replace(/[^\d\+]/g,'');
+  if (!phone || phone.length === 0) return '';
 
-  if (phone.substring(0, 1) == "+") {
-    phone = "+" + phone.replace(/[^\d]/g,'');
-  } else {
-    phone = phone.replace(/[^\d]/g,'');
-  }
+  var hasPlus = false;
+  if (phone.charAt(0) === "+") hasPlus = true;
+
+  phone = phone.replace(/[^\d]/g,'');
 
-  return phone;
+  return hasPlus ? '+' + phone : phone;
```

Comment:
A phone number with the syntax `+15555555555` would be a valid number, but we require them to be strings in all of our use cases